scripts/functions/help_system

```diff
@@ -15,11 +15,11 @@ function print_arrays()
     ##               "-a does this"\
     ##               "-b does that")
     ##
-    ## Note: Line Continuations "\" are _required_ when building multi-argument help systems
+    ## Note: Line Skips Continuations "\" are _required_ when building multi-argument help systems
```

Comment:
so would this be ok? 

"-a does this"\"-b does that"

if not,
this is what I meant by the "skip and the continuation \" are 'both' required

Reply:
I can make that change... but I don't understand it. This is allowed in bash:

```bash
moose-test_opt -i something.i \
\
--more-options \
--even-more-options
```

I am not sure where "Skip" comes into this. Unless what I demonstrated above is what you meant by a line continuation _and_ and 'skip'.

Reply:
I m saying if you put everything on the same line it does not work does it. '-a -b' and ''argument a description argument b description"

so the line break is necessary just as much as "\"